kclvm/sema/src/resolver/import.rs

```diff
@@ -98,10 +100,29 @@ impl<'ctx> Resolver<'ctx> {
                             {
                                 match self.ctx.import_names.get_mut(&self.ctx.filename) {
                                     Some(mapping) => {
-                                        mapping.insert(
-                                            import_stmt.name.to_string(),
-                                            import_stmt.path.to_string(),
-                                        );
+                                        if mapping.contains_key(&import_stmt.name.to_string()) {
+                                            self.handler.add_warning(
+                                                WarningKind::ReimportWarning,
+                                                &[Message {
+                                                    pos: Position {
+                                                        filename: module.filename.clone(),
+                                                        line: stmt.line,
+                                                        column: Some(0),
+                                                    },
+                                                    style: Style::Line,
+                                                    message: format!(
+                                                        "{} is reimported multiple times.",
```

Comment:
module '{}' is reimported multiple times.

Reply:
has fixed